compiler/rustc_metadata/src/creader.rs

```diff
@@ -1132,6 +1132,12 @@ fn load_dylib(path: &Path, max_attempts: usize) -> Result<libloading::Library, S
             Err(err) => {
                 // Only try to recover from this specific error.
                 if !matches!(err, libloading::Error::LoadLibraryExW { .. }) {
+                    let err = err.to_string();
```

Comment:
This should use `format_dlopen_err` too I think. And then below  it should strip `format(": {}", path.display())`.

Reply:
Good idea!